json-schema/schema.json

```diff
@@ -460,7 +465,18 @@
       "examples": [
         "torch.save",
         "torch.jit.save",
-        "torch.export.save"
+        "torch.export.save",
+        "tf.keras.Model.save",
+        "tf.keras.Model.save_weights",
+        "tf.saved_model.export(format='tf_saved_model')"
+      ]
+    },
+    "mlm:compile_method": {
+      "type": "string",
+      "minLength": 1,
```

Comment:
yes that's the intention. I think it'd be burdensome to require setting null explicitly here.

Reply:
Ok great. I suggest the set of values then omits the `null`, and that is mentioned only in the type as `Compile Method | null`. This will match the representation used by other fields.